open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/handlers/ProcessContextHandler.java

```diff
@@ -78,6 +78,10 @@ public ProcessContextHandler(String serviceName,
             Optional<EntityDetail> entityDetail = commonHandler.getEntityDetails(userId, processGuid, PROCESS);
             if (!entityDetail.isPresent()) {
                 log.error("Entity with guid {} was not found in any metadata repository", processGuid);
+//                AssetLineageErrorCode errorCode = AssetLineageErrorCode.ENTITY_NOT_FOUND;
```

Comment:
You should remove the commented code if it doesn't make sense to have it anymore